test/shared_test/lib_battery_properties.h

```diff
@@ -89,14 +89,13 @@ class BatteryProperties : public ::testing::Test
 
 		// thermal
 		mass = 507;
-		length = 0.58;
-		width = 0.58;
-		height = 0.58;
+		length = 1.1;
```

Comment:
why did these input values change?

Reply:
Same reason as above. I don't know any good estimates for thermal parameteres but this (paper)[https://iopscience.iop.org/article/10.1088/1757-899X/53/1/012014/pdf] provides an example set of parameters where the ratio of mass to surface area is lower than provided here originally

![Screen Shot 2019-10-09 at 4 00 38 PM](https://user-images.githubusercontent.com/29387736/66523783-fc9e8600-eaad-11e9-8227-a1760c1d50f8.png)